src/com/facebook/buck/python/PythonBuckConfig.java

```diff
@@ -286,6 +301,7 @@ public PackageStyle getPackageStyle() {
   public enum PackageStyle {
     STANDALONE,
     INPLACE,
+    PEX_INPLACE,
```

Comment:
Long-term we'd like to move all the PEX-specific naming/code out of Buck, just supporting the general way to use arbitrary custom Python packagers via the `python:path_to_pex` config setting.

In this vein, it'd be nice to at least keep the zip-vs-directory choice separate from the inplace-vs-standalone choice. What do you think about having a separate PexStyle enum with `ZIP` and `DIRECTORY` choices (which would apply when `STANDALONE` is used with the default PEX builder)?

Another option might be to make the current in-place package style more general and provide a config option to choose between "simple" and "PEX" sub-styles, the latter of which implements the in-place PEX. The annoying thing about this is that the current code path for in-place assumes that the generated symlink trees with the modules will be used and that the in-place starter script won't do any of it's own packaging (even if it is just copying to a directory).